tests/system/providers/apache/kafka/example_dag_event_listener.py

```diff
@@ -26,12 +26,12 @@
 
 # Connections needed for this example dag to finish
 from airflow.models import Connection
+from airflow.providers.apache.kafka.operators.produce import ProduceToTopicOperator
+from airflow.providers.apache.kafka.sensors.kafka import AwaitMessageTriggerFunctionSensor
 
 # This is just for setting up connections in the demo - you should use standard
 # methods for setting these connections in production
-from airflow.operators.python import PythonOperator
-from airflow.providers.apache.kafka.operators.produce import ProduceToTopicOperator
-from airflow.providers.apache.kafka.sensors.kafka import AwaitMessageTriggerFunctionSensor
+from airflow.providers.standard.core.operators.python import PythonOperator
```

Comment:
Python and Bash operators are currently used in several system tests as well as tests for other providers, do we ignore doing refactoring system test imports until standard provider releases, or any other approach to work these imports?

Reply:
or should i add something like this 
```
try:
    from airflow.providers.standard.core.operators.python import PythonOperator
except ImportError:
    from airflow.operators.python import PythonOperator
```

Reply:
This is not required imports. it is working without try catch.